settings.gradle

```diff
@@ -10,6 +10,7 @@ include 'py'
 include 'repl'
 include 'docker'
 include 'app-streaming'
+include 'package'
```

Comment:
Just make sure, it is published.

Reply:
yup, I checked ant the artefact was fine. Also was testing a lot using maven central and also using Jitpack ( It can be used as maven repository as well ) using `--packages`, all was good.

Will verify once again after release.